app/src/main/java/app/gamenative/ui/screen/library/appscreen/SteamAppScreen.kt

```
@@ -222,18 +222,18 @@ class SteamAppScreen : BaseAppScreen() {
+            try {
+                val containerManager = ContainerManager(context)
+                val container = containerManager.containers.find { it.id == libraryItem.appId }
+                if (container != null && container.lastPlayedTimestamp > 0) {
+                    SteamUtils.fromSteamTime((container.lastPlayedTimestamp / 1000).toInt())
                 } else {
-                    context.getString(R.string.steam_never)
+                    context.getString(R.string.library_never_played)
                 }
-            } else {
-                context.getString(R.string.steam_never)
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)